### ISSUE TITLE
add docs for the default array type conversion?

### ISSUE THREAD

**Issue by ianstormtaylor** [brianc/node-postgres#1462]
Thanks for a great library!

Looking through the issues, it seems like arrays are converted to Postgres arrays. But I wasn't able to find a clear answer to whether arrays of objects are converted to JSON arrays or Postgres arrays. Either way, whatever the answer ends up being could it be added to the type docs for future users?

**Comment by abenhamdine** [comment]
Yes good idea to add to the docs, though it's seems more related to https://github.com/brianc/node-pg-types

See also https://github.com/brianc/node-postgres/pull/1432 (not merged) for more background.

**Comment by ianstormtaylor** [comment]
Awesome, thanks @abenhamdine that PR explains it well, and the decision makes sense!
